dt-assets/js/comments.js

```diff
@@ -136,7 +136,15 @@ jQuery(document).ready(function ($) {
         if (a.comment){ %>
           <% is_Comment = true; %>
             <div dir="auto" class="comment-bubble <%- a.comment_ID %>" data-comment-id="<%- a.comment_ID %>">
-              <div class="comment-text" title="<%- a.date_formatted %>" dir=auto><%= a.text.replace(/\\n/g, '</div><div class="comment-text" dir=auto>') /* not escaped on purpose */ %></div>
+				<div class="comment-text" title="<%- a.date_formatted %>" dir=auto>
+    <% 
+    if (a.text.indexOf('<li>') === -1) {
+    %>
+        <%= a.text.replace(/\\n/g, '</div><div class="comment-text" dir=auto>') /* not escaped on purpose */ %>
+    <% } else { %>
+        <%= a.text %>
+    <% } %>
+</div>
```

Comment:
@CptHappyHands please explain this change.

Reply:
This part was to make the lists function properly. When I was trying to add the lists, it would get replaced by the a.text.replace on line 143, and wouldn't look correct. So, decided to check if it was a list on line 141, and if it wasn't, proceed as it was previously, but if it was a list, just return a.text on 145 which returned the <li>